### ISSUE TITLE
Is it possible to show the increase or decrease of a stock?

### ISSUE THREAD

**Issue by nkta3m** [gaborvecsei/Stocks-Pattern-Analyzer#13]
Hi guys 
Is it possible that the line chart presented in this project shows the number of increases or decreases?

Thank you

**Comment by gaborvecsei** [comment]
@nkta3m, yes, this could be easily achieved with improving the plotting a bit. If you create a PR I will check it out as soon as I can. 🙂
